command/pr_review.go

```diff
@@ -125,10 +133,123 @@ func prReview(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if input == nil {
+		input, err = reviewSurvey(cmd)
+		if err != nil {
+			return err
+		}
+		if input == nil && err == nil {
+			// Cancelled.
+			return nil
+		}
+	}
+
 	err = api.AddReview(apiClient, pr, input)
 	if err != nil {
 		return fmt.Errorf("failed to create review: %w", err)
 	}
 
 	return nil
 }
+
+func reviewSurvey(cmd *cobra.Command) (*api.PullRequestReviewInput, error) {
+	editorCommand, err := determineEditor(cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	typeAnswers := struct {
+		ReviewType string
+	}{}
+	typeQs := []*survey.Question{
+		{
+			Name: "reviewType",
+			Prompt: &survey.Select{
+				Message: "What kind of review do you want to create?",
+				Options: []string{
+					"Comment",
+					"Approve",
+					"Request Changes",
```

Comment:
```suggestion
					"Request changes",
```
Been trying to stick to sentence case like GitHub.com :)